app/models.py

```diff
@@ -110,15 +110,21 @@ def send_message(self, text, recipient_room):
 
     def upload_photo(self, photo):
         if self.photo != photo:
-            filename = str(self.id) + "." + photo.data.filename.rsplit('.', 1)[1]
-
-            file_path = os.path.join(Constants.IMAGE_UPLOAD_FOLDER, filename)
-            file_path_db = os.path.join(Constants.IMAGE_DB_FOLDER, filename)
+            # Delete old photo
+            old_path = os.path.join(Constants.IMAGE_UPLOAD_FOLDER, os.path.basename(self.photo))
+            if os.path.isfile(old_path):
+                os.unlink(old_path)
 
+            # Crate folder for photos
             if not os.path.isdir(Constants.IMAGE_UPLOAD_FOLDER):
                 os.makedirs(Constants.IMAGE_UPLOAD_FOLDER)
```

Comment:
Seems to be better to create such folders in create_app()

Reply:
Fixed